bin/reth/src/config.rs

```diff
@@ -78,3 +83,12 @@ impl Default for SenderRecoveryConfig {
         Self { commit_threshold: 5_000, batch_size: 1000 }
     }
 }
+
+/// Configuration for each stage in the pipeline.
```

Comment:
Should be specifically about peers Config not for every stage. And yes if we can dedup with other definitions let's try that.